src/Workspaces/Core/MSBuild.BuildHost/IBuildHost.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost;
 internal interface IBuildHost
 {
     /// <summary>
-    /// Returns whether this project's language is supported.
+    /// Returns whether this project's is supported by this host, considering both the project language and also MSBuild availability.
```

Comment:
I initially thought just to add a separate method here, but it's a bit strange since in the .NET Core we case we still use the project file path as an input to the MSBuild discovery too. The "proper" fix here is to refactor MSBuildLocator to let us discover the (lack of) VS installed in this case in proc, but that'll require MSBuildLocator changes.

Reply:
we should have a tracking issue for all the changes we want to make to msbuild locator - we have this one and I think there was another one you mentioned before.